devfile.yaml

```diff
@@ -17,7 +17,7 @@ metadata:
 components:
   - name: tooling-container
     container:
-      image: quay.io/devfile/universal-developer-image:ubi8-latest
+      image: quay.io/mkuznets/che-operator-tools:latest
```

Comment:
Personal quay repo should be avoided. Something like quay.io/eclipse/che-operator-dev would be better

Reply:
```suggestion
      image: quay.io/eclipse/che-operator-dev:latest
```